src/cargo/util/toml/targets.rs

```diff
@@ -176,12 +176,12 @@ pub fn resolve_lib(
 #[tracing::instrument(skip_all)]
 fn to_lib_target(
     original_lib: Option<&TomlLibTarget>,
-    resolved_lib: Option<&TomlLibTarget>,
+    normalized_lib: Option<&TomlLibTarget>,
     package_root: &Path,
     edition: Edition,
     warnings: &mut Vec<String>,
 ) -> CargoResult<Option<Target>> {
-    let Some(lib) = resolved_lib else {
+    let Some(lib) = normalized_lib else {
         return Ok(None);
     };
 
```

Comment:
Some other locations need update are out of the diff:

* `fn is_resolved`
* `expect("previously resovled")` and `expect("should be resolved")`.